### PR TITLE
{CI} Add `time.sleep` to ensure that the live test can delete the resource group

### PR DESCRIPTION
**Pull request by wangzelin007**
**Related command**

**Description**

During the 'webapp vnet-integration remove' call, we enter 20s wait for hit geo and 45s wait on app service side in order for NCs to be deleted successfully on machines.
So add time.sleep(65) for all `az webapp vnet-integration` related commands.
This can avoid resource residue caused by appservice related tests.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

File: src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py [hunk 1]
```diff
@@ -2206,11 +2206,13 @@ def test_webapp_vnetE2E(self, resource_group):
         ])
         self.cmd(
             'webapp vnet-integration remove -g {} -n {}'.format(resource_group, webapp_name))
+        time.sleep(65)
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py:2209]: I'm curious the reason that we sleep `65`.

Comment (wangzelin007) [reply src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py:2209]: During the 'webapp vnet-integration remove' call, we enter 20s wait for hit geo and 45s wait on app service side in order for NCs to be deleted successfully on machines.
So add time.sleep(65) for all az webapp vnet-integration related commands.

**Review by bebound — DISMISSED** [review]

File: src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py (hunk 1 above)
Comment (jsntcy) [reply src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py:2209]: Then define a constant for `65` and add the comments you provided above for the constant.

**Review by jsntcy — APPROVED** [review]

**Review by jsntcy — APPROVED** [review]

**Pull request merged**